contentful/content-migrations/migrations/20241128-1400-add-validation-to-short-text-fields.js

```diff
@@ -0,0 +1,95 @@
+const freeTextFields = {
```

Comment:
Could this be done programatically as well? Maybe with _exclusions_ instead of _inclusions_ for the fields? Just thinking it might be good to have standard validation on freeform text fields, that we can easily implement at any time.

Reply:
With the migration library, no, but we could start the work on combining the JS projects and do it programatically - however theres another problem, the management library doesn't respect capitalisation, yet requires it for the migration library 😭 so the fieldIds were coming back as "slug" for instance, but must be specified as "Slug" for the migration library to work... so I had to amend these manually... 😢